### PR TITLE
feat(admin-gateway): Begin using pmg-device-service and digital-session-services's retrofit clients

### PR DESCRIPTION
**Pull request by timrs2998**
digital-session isn't quite working since Retrofit can't instantiate `ResponseEntity<LastKnownLocation>` from the `UIController`

Mocked out controllers that I expect to add (from looking at admin-portal).

### CODE REVIEW

File: admin-gateway-service.gradle [hunk 1]
```diff
@@ -12,6 +12,7 @@ buildscript {
     }
```
Comment (aaronzirbes) [inline admin-gateway-service.gradle]: you can bump to `com.peoplenet:gradle-plugins:0.6.0'` now

File: admin-gateway-service.gradle [hunk 2]
```diff
@@ -23,22 +24,17 @@ run {
     jvmArgs '-Dspring.profiles.active=dev', '-Duser.timezone=UTC'
 }
 
-build.doFirst {
```
Comment (aaronzirbes) [inline admin-gateway-service.gradle]: I don't think it's safe to delete this

File: src/main/resources/application.yml [hunk 3]
```diff
@@ -17,8 +17,7 @@ healthcheck:
 networkaddress:
   cache:
     ttl: 60
-logging.level:
-    com.peoplenet.retrofit.client: ERROR
+logging.level.com.peoplenet: ERROR
```
Comment (aaronzirbes) [inline src/main/resources/application.yml]: shouldn't this be `WARN`? or is that too noisy?

File: src/test/resources/application.yml [hunk 4]
```diff
@@ -17,5 +17,4 @@ healthcheck:
 networkaddress:
   cache:
     ttl: 60
-logging.level:
-    com.peoplenet.retrofit.client: ERROR
+logging.level.com.peoplenet: INFO
```
Comment (aaronzirbes) [inline src/test/resources/application.yml]: I like setting the test logger to DEBUG

File: src/main/resources/application.yml (hunk 3 above)
Comment (timrs2998) [inline src/main/resources/application.yml]: Logging currently doesn't work for me in this project yet. Once spring starts up, nothing new gets logged.

Comment (aaronzirbes) [inline src/main/resources/application.yml]: You might be missing the slf4j libraries you need.

You should get that through `org.springframework.boot:spring-boot-starter-actuator:1.2.3.RELEASE` thouhg

**Comment by timrs2998** [comment]
:warning: Will work on logging now

**Pull request closed without merging**